examples/obs-csi-plugin/kubernetes/bucket/pvc.yaml

```diff
@@ -0,0 +1,11 @@
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: pvc-obs
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 500Gi
```

Comment:
This parameter is redundant, please optimize and remove it later.

Reply:
done

Reply:
The PersistentVolumeClaim "pvc-obs" is invalid: spec.resources[storage]: Required value
this parameter is required for pvc, and must be greater than zero